addons/mail/views/mail_activity_views.xml

```diff
@@ -203,6 +227,18 @@
         </field>
     </record>
 
+    <record id="mail_activity_action_tree" model="ir.actions.act_window.view">
```

Comment:
Guidelines: action_view_tree (that way we know this is an act_window.view record)

Reply:
Done.